### ISSUE TITLE
Prebuilt WoA Images??

### ISSUE THREAD

**Issue by moltenteam** [andreiw/RaspberryPiPkg#94]
Hello, can somebody provide me some WoA images? I tried to built mine on uup.rg-adguard.net, but my antivirus (Avast Antivirus) was blocking the ISOCompiler out. Thanks for any replies

**Comment by BenNottelling** [comment]
Add it as an exception..

**Comment by moltenteam** [comment]
Yeah, it was behavioural detection. I was really stupid. Thank you @BenNotteling

**Comment by BenNottelling** [comment]
No problem @moltenteam, sorry if my comment came off as rude!
